gradle.properties

```diff
@@ -19,6 +19,8 @@ httpClientVersion=4.5.1
 jedisVersion=2.8.1
 h2Version=1.4.191
 springDataMongoVersion=1.9.1.RELEASE
+springDataCassandraVersion=1.5.0.M1
```

Comment:
Is Ingalls milestone release really required here? Other Spring Data dependencies are on Hopper release train level ATM, so we should use `1.4.2.RELEASE`.